api/ServiceWorkerRegistration.json

```diff
@@ -1259,6 +1259,71 @@
         }
       },
       "updateViaCache": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ServiceWorkerRegistration/updateViaCache",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "56",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "Service Worker payment apps",
+                  "value_to_set": "Enabled"
+                }
+              ]
+            },
+            "chrome_android": {
+              "version_added": "56",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "Service Worker payment apps",
+                  "value_to_set": "Enabled"
+                }
+              ]
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": null
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "paymentManager": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/ServiceWorkerRegistration/updateViaCache",
```

Comment:
I spotted this error, and then noticed a number of other mistakes in this file:

* line 759 - "showNotification" should be "action"
* line 810 - "showNotification" should be "badge"
* line 861 - "showNotification" should be "data"
* line 912 - "showNotification" should be "image"
* line 963 - "showNotification" should be "renotify"
* line 1014 - "showNotification" should be "requireInteraction"
* line 1065 - "showNotification" should be "vibrate"
* line 1328 - "updateViaCache" should be "paymentManager"